addons/patches/optre_covenant_tweaks/CfgVehicles.inc

```diff
@@ -0,0 +1,116 @@
+class CfgVehicles
+{
+	class StaticMGWeapon;
+	class AAA_System_01_base_F: StaticMGWeapon
```

Comment:
Remove everything in this class and titlecase, e.g.
```hpp
class AAA_System_01_Base_F;
```

Then you should be good.